### DISCUSSION TITLE
Implementation of a ready project

### DISCUSSION THREAD

**Discussion by Asbaharoon** [community/community#20655] [Discussions] [answered]
Good Morning Dears 

i like some Javafx project for bank management and i would like to use it as test, So how i run it in application and is NetBeans application is enough or must be continue to study programming from beginning ???,

**Comment by karafra** [comment, marked as answer]
> Good Morning Dears 
> 
> i like some Javafx project for bank management and i would like to use it as test, So how i run it in application and is NetBeans application is enough or must be continue to study programming from beginning ???,   

Could you rephrase? I dont understand you.

**Reply by Asbaharoon** [reply]
Sorry my friend and i will try again in application after import codes and convert to **Jar** file
